schema_based_on_diagram.sql

```diff
@@ -38,9 +38,17 @@ create table invoice_items(
     foreign key (treatment_id) REFERENCES treatments(id)
 )
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221996885-742699a1-6883-4ad6-86f9-4d3653ffa845.png)

**[Optional]**

- [ ] Kindly add a message in this section such as **"If you like this project then don't forget to give a :star: star on this repository"** to make your readme file descriptive and professional :+1: